### ISSUE TITLE
this.listFormatter.match is not a function

### ISSUE THREAD

**Issue by QXD-me** [ng2-ui/ng2-auto-complete#168]
I have been getting the error "this.listFormatter.match is not a function" in version 0.11.0, but didn't get it in version 0.10.12. The list-formatter I'm referring to is a function.
e.g.

```
<ng2-auto-complete
   ...
[list-formatter]="autocompleListFormatter"
... />
```

```
public autocompleListFormatter = (data: MyClass): string => {

        return (data.description == undefined || data.description.length === 0)
			? data.code
        : `${data.code} (${data.description})`;
}

```

**Comment by allenhwkim** [comment]
@QXD-me thanks for reporting this. Fixed and released as 0.11.1

**Comment by QXD-me** [comment]
Thanks @allenhwkim . I just updated and it seems to work fine.
